### PR TITLE
AMLS-5479 | Changing AMP models for differ in content

### PR DESCRIPTION
**Pull request by matthewharris1**
As above. Models needed changed due to changes to file name and Json names in AMP.

DON'T MERGE - Until AMP branch is also merged as they depend on each other

## How Has This Been Tested?
Unit
Manual

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [ ] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/models/amp/Amp.scala [hunk 1]
```diff
@@ -59,10 +59,10 @@ final case class Amp(data: JsObject = Json.obj(),
   }
 
   private def isBoughtOrSoldOverThresholdComplete: Boolean = {
```
Comment (slavvco) [inline app/models/amp/Amp.scala]: isBought?

File: test/models/amp/AmpSpec.scala [hunk 2]
```diff
@@ -98,7 +98,7 @@ trait AmpValues {
   val completeModel                             = Amp(completeData, false, true)
   val missingTypeOfParticipantModel             = Amp(missingTypeOfParticipantData)
   val missingTypeOfParticipantDetailModel       = Amp(missingTypeOfParticipantDetailData)
-  val missingBoughtOrSoldOverTheThresholdModel  = Amp(missingBoughtOrSoldOverThresholdData)
+  val missingBoughtOrSoldOverTheThresholdModel  = Amp(missingSoldOverThresholdData)
```
Comment (slavvco) [inline test/models/amp/AmpSpec.scala]: missingBought?

File: app/models/amp/Amp.scala (hunk 1 above)
Comment (matthewharris1) [reply app/models/amp/Amp.scala]: Done

File: test/models/amp/AmpSpec.scala (hunk 2 above)
Comment (matthewharris1) [reply test/models/amp/AmpSpec.scala]: Done

**Pull request merged**